src/pages/html/tutorials/how-to-create-an-html-button-that-acts-like-a-link/index.md

```diff
@@ -3,13 +3,46 @@ title: How to Create an HTML Button That Acts Like a Link
 ---
 ## How to Create an HTML Button That Acts Like a Link
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/html/tutorials/how-to-create-an-html-button-that-acts-like-a-link/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+This guide will demonstrate "How to Create an HTML Button That Acts Like a Link".
+Programmers and Web Developers use buttons for several different ocassions. 
+Today you will create your very own social link button! 
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+## Step 1 Button Tag
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+No not a HashTag, but a button tag. This is the first step and it is the step that defines to the computer
+what you want to see. Lets Start!
+
+<button>
+  
+# BEST Practice
+
+You should get in the habit of closing your tags first before writing code. So as to not run into unwanted errors or your code just not working at all. It may seem silly but when you have alot of code written down it can be easy to forget to close your tags or divs. Resulting in bad or not working code.
+
+So lets close the button tag, like so...
```

Comment:
Try to avoid talking like this, as it is not consistent with the style of the guide.